src/Lender.sol

```diff
@@ -499,6 +464,12 @@ contract Lender {
         return true;
     }
 
+    /// @notice allows admin to add a protocol contract for reference by adapters
+    /// @param a address of a new protocol contract
+    function addContract(address a) external authorized(admin) {
```

Comment:
I know we have to be generic, because we add various contracts...

But "addContract" seems a bit too generic, maybe we can think of a slightly more precise naming

`addPeriphery` or hmm im thinking of something other than just "contract"

Reply:
That's a fair point. I think the right wording might be to call it `addRouter` as we typically hook into individual protocol's routers. I'll make the change.